addon/README.md

```diff
@@ -1,134 +1,138 @@
-
-For more general details on where these metric events are sent for measurement &
-analysis, read [`../docs/metrics/index.md`](../docs/metrics/index.md).
-
-## Participation Indicators
-
-For certain whitelisted URLs - currently just `about:home` and `mozilla.com` sites and Test Pilot itself - these properties are injected into the window:
-
-* `window.navigator.testpilotAddon = true` - indicates Test Pilot participation
-* `window.navigator.testpilotAddonVersion = 2.0` - or current add-on version
-
-Additionally, for the Test Pilot site, this property is included:
-
-* `window.navigator.testpilotClientUUID = {unique id}` - a unique ID generated when the Test Pilot add-on is installed for metrics purposes
-
-Interesting source files for this feature include the following:
-
-* [`./src/chrome/scripts/frame-script.js`](./src/chrome/scripts/frame-script.js) - content process frame script that interacts with pages to inject the properties.
-* [`./src/lib/frameScripts.js`](./src/lib/frameScripts.js) - Bootstrap-side module that injects the frame script into windows and relays property changes from the WebExtension side.
-* [`./src/lib/webextension/environments.js`](./src/lib/webextension/environments.js) - WebExtension-side module that determines which Test Pilot site environment is active, defines the whitelist of sites for property injection.
-* [`./src/lib/webextension/metrics.js`](./src/lib/webextension/metrics.js) - WebExtension-side module that generates the unique client UUID for metrics purposes.
+Test Pilot main add-on
+----------------------
+
+This add-on serves a few basic purposes common to participation in any Test Pilot experiment:
+
+* Toolbar button as a quick shortcut to Test Pilot.
+* Notifications when experiments are updated and new ones arrive.
+* General metrics reporting about participation in experiments and Test Pilot in general.
+* Telemetry metrics support for bootstrap & WebExtension based experiments.
+* Signals to about:home and mozilla.com websites indicating participation in Test Pilot.
+
+## Building and Development
+
+First, ensure that you've followed [the quickstart guide](../docs/development/quickstart.md) for working on the Test Pilot site itself. This includes ensuring you've got the right version of Node.js (currently, [v6.x LTS](https://nodejs.org/dist/latest-v6.x/)).
+
+From there, you'll need to install dependencies specific to the add-on:
+```
+cd addon
+npm install
+```
+
+If you just want an XPI package of the add-on, use one of these commands:
+```
+npm package      # Production mode without logging
+npm package:dev  # Development mode with lots of Browser Console logging
+```
+
+If you'd like to actively work on the add-on, here are some additional steps to set up a more convenient workflow:
+
+1. Install [Firefox Developer Edition][devedition]. (Nightly should work, too, but Dev Edition is preferred.)
+
+1. Set `extensions.legacy.enabled` to `true` [using `about:config`][aboutconfig],
```

Comment:
Huh. Not sure why the diff looks like I re-wrote the whole file. Adding this bullet about extensions.legacy.enabled is all I changed.